pkg/i18n/english.go

```diff
@@ -1143,6 +1143,9 @@ func addEnglish(i18nObject *i18n.Bundle) error {
 		}, &i18n.Message{
 			ID:    "copyCommitShaToClipboard",
 			Other: "copy commit SHA to clipboard",
+		}, &i18n.Message{
+			ID:    "copyCommitMessageToClipboard",
```

Comment:
if you rebase onto master you'll find that this file is a bit different, but it's still fairly straightforward: you'll want a new field on the `TranslationSet` struct called `LcCopyCommitMessageToClipboard` (lc stands for lowercase because golang treats lower-case field names as private) and then you'll want to return the value in `englishTranslationSet` (you can follow along with what we do with other translations in that file)